compatibility/bazel_tools/testing.bzl

```diff
@@ -855,6 +855,18 @@ excluded_test_tool_tests = [
             },
         ],
     },
+    {
+        "start": "2.6.0-snapshot.20221226.11190.0.71548477",
+        "platform_ranges": [
+            {
+                "end": "2.6.0-snapshot.20221226.11190.0.71548477",
+                "exclusions": [
+                    "ContractLKeyIT:CKLocalLookupByKeyVisibility",
+                    "ContractLKeyIT:CKLocalFetchByKeyVisibility",
+                ],
+            },
+        ],
+    },
```

Comment:
Thanks for this. It's something that would have taken forever for myself to uncover!

Reply:
You'r welcomed